spec/features/likes_feature_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+feature 'Likes' do
+
+  before { Picture.create title: 'My picture' }
+
+  scenario 'a user can like a picture, which will increase "likes" count', js: true do
```

Comment:
Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)